client-ts/signalr/src/Transports.ts

```diff
@@ -29,21 +29,21 @@ export interface ITransport {
 
 export class WebSocketTransport implements ITransport {
     private readonly logger: ILogger;
-    private readonly accessToken: () => string;
+    private readonly accessTokenFactory: () => string;
     private webSocket: WebSocket;
 
-    constructor(accessToken: () => string, logger: ILogger) {
+    constructor(accessTokenFactory: () => string, logger: ILogger) {
         this.logger = logger;
-        this.accessToken = accessToken;
+        this.accessTokenFactory = accessTokenFactory;
     }
 
     connect(url: string, requestedTransferMode: TransferMode, connection: IConnection): Promise<TransferMode> {
 
         return new Promise<TransferMode>((resolve, reject) => {
             url = url.replace(/^http/, "ws");
-            if (this.accessToken) {
-                let token = this.accessToken();
-                url += (url.indexOf("?") < 0 ? "?" : "&") + `signalRTokenHeader=${token}`;
+            if (this.accessTokenFactory) {
+                let token = this.accessTokenFactory();
+                url += (url.indexOf("?") < 0 ? "?" : "&") + `access_token=${token}`;
```

Comment:
Same question (also, should you URLencode the token query string parameter, just in case?)